tests/test_keyboard_shortcuts.py

```diff
@@ -0,0 +1,279 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import random
+
+from pages.treeherder import TreeherderPage
+
+
+def test_close_open_panels(base_url, selenium):
+    """Shortcut: 'esc'
+    Open Treeherder page, open Filters panel, select random job, close all
+    panels using 'esc' button, verify if all panels are closed"""
+    page = TreeherderPage(selenium, base_url).open()
+
+    page.click_on_filters_panel()
+    page.select_random_job()
+
+    assert page.filter_panel_is_opened
+    assert page.job_details_panel_is_opened
+
+    page.close_all_panels()
+
+    assert not page.filter_panel_is_opened
+    assert not page.job_details_panel_is_opened
+
+
+def test_enter_quick_filter_shortcut(base_url, selenium):
+    """Shortcut: 'f'
+    Open Treeherder page, verify if search box is empty, enter search box
+    filter using 'f' shortcut, type 'mozilla', verify if filter box contain
+    word mozilla"""
+    page = TreeherderPage(selenium, base_url).open()
+    assert page.get_search_box_text == ''
+
+    page.filter_by_using_quick_filter('mozilla')
+
+    assert page.get_search_box_text == 'mozilla'
+
+
+def test_clear_the_quick_filter_shortcut(base_url, selenium):
+    """Shortcut: CTRL + SHIFT + 'f'
+    Open Treeherder page, filter by 'mozilla', verify if filter box contain
+    word 'mozilla', clear the quick filter using CTRL + SHIFT + f shortcut,
+    verify if search box is empty"""
+    page = TreeherderPage(selenium, base_url).open()
+
+    page.filter_by_using_quick_filter('mozilla')
+    assert page.get_search_box_text == 'mozilla'
+
+    page.clear_filter_by_shortcut()
+
+    assert page.get_search_box_text == ''
+
+
+def test_next_job_shortcut(base_url, selenium):
+    """Shortcut: 'Rigth Arrow'
+    Open Treeherder page, select random job, select job next to it and take
+    job keyword, go back to previous job and select next job using Right arrow
+    shortcut, verify if job keyword match"""
+    page = TreeherderPage(selenium, base_url).open()
+    all_jobs = page.all_jobs
+
+    # Check number of jobs
+    num_of_jobs = len(all_jobs) - 1
+    rnd_number = random.randint(0, num_of_jobs)
+    next_job = rnd_number + 1
+
+    if rnd_number == num_of_jobs:
+        next_job = 0
```

Comment:
You're already deducting 1 from the number of jobs, so number of jobs + 1 would be the last job, but an out of range error. I think it's better to do this unless we specifically want to test boundaries.